blocks/init/src/Blocks/components/menu/assets/main-menu-a11y.js

```diff
@@ -0,0 +1,109 @@
+export class AccessibleMenu {
+	constructor(options) {
+		this.menuSelector = options.menuSelector;
+		this.anchor = options.anchor;
+		this.subMenu = options.subMenu;
+
+		this.mainMenu = document.querySelector(this.menuSelector);
+		this.links = this.mainMenu.querySelectorAll(this.anchor);
+
+		this.IS_FOCUSED_CLASS = 'is-focused';
+	}
+
+	addClassName(el, cls) {
+		if (!el.className.match(`(?:^|\\s)${cls}(?!\\S)`)) {
+			el.className += ` ${cls}`;
+		}
+	}
+
+	hasClassName(el, cls) {
+		return !!el.className.match(`(?:^|\\s)${cls}(?!\\S)`);
+	}
+
+	delClassName(el, cls) {
+		el.className = el.className.replace(new RegExp(`(?:^|\\s)${cls}(?!\\S)`), '');
+	}
+
+	tabCloseMenu() {
+		const navActive = this.mainMenu.querySelectorAll(`.${this.IS_FOCUSED_CLASS}`);
```

Comment:
Shouldn't this be `this.mainMenu.querySelector(`.${this.IS_FOCUSED_CLASS}`);`? (since there can only be 1 thing that is focused at a time).

Reply:
🤔  That actually makes sense 👍🏼